tests/urls.py

```diff
@@ -1,10 +1,18 @@
+from django.conf import settings
 from django.conf.urls import include, url
+from django.conf.urls.static import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
 from openwisp_utils.admin_theme.admin import admin, openwisp_admin
 
+
 openwisp_admin()
 admin.autodiscover()
 
 urlpatterns = [
     url(r'^', include('django_ipam.urls', namespace='ipam')),
     url(r'^admin/', admin.site.urls),
 ]
+
+urlpatterns += staticfiles_urlpatterns()
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
not sure this addition is needed, but let's keep it on hold until we resolve the doubts